Packs/AtlassianConfluenceCloud/Integrations/AtlassianConfluenceCloud/AtlassianConfluenceCloud.py

```diff
@@ -965,6 +965,19 @@ def validate_update_content_args(args: dict[str, str]):
         raise ValueError(MESSAGES["REQUIRED_ARGUMENT"].format("version"))
 
 
+def validate_get_content_args(args: dict[str, str]):
+    """
+    Validate arguments for confluence-cloud-content-get command, raise ValueError on invalid arguments.
+
+    :type args: ``Dict[str, str]``
+    :param args: The command arguments provided by the user.
+
+    :return: None
+    """
+    content_id = args["content_id"]
+    if not content_id:
+        raise ValueError(MESSAGES["REQUIRED_ARGUMENT"].format("content_id"))
+
```

Comment:
no need to add this validation, just add the key "required: true" to the arg in the yml file